### PR TITLE
Add apoc recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@haesleinhuepf do you agree with being added as a maintainer?

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17725

**Comment by haesleinhuepf** [comment]
> @haesleinhuepf, do you agree with being added as a maintainer?

Yes, I agree. Thanks for working on this!

**Comment by goanpeca** [comment]
>Recipe with the same name exists in bioconda: please discuss with https://github.com/orgs/conda-forge/teams/bioconda-recipes.

Should we change the name of the package @jaimergp ? It is not the same package, but they have the same name on bioconda

**Comment by jaimergp** [comment]
@goanpeca, yes, definitely. Maybe something like `apoc-backend`?

**Comment by jaimergp** [comment]
On the other hand, this one has the namespace on PyPI and the one on bioconda has been [stale for a while](https://github.com/bioconda/bioconda-recipes/blame/076c5aa81e36b8bf875ebff07f330b2b04f40ce5/recipes/apoc/meta.yaml#L3) (5 years) and the linked [website](https://cssb.biology.gatech.edu/skolnick/files/APoc/) is no longer online (it's [here](https://sites.gatech.edu/cssb/APoc/) though). 

I think it's fine to let this one take `apoc` here. It _might_ break people's environments if they are using both conda-forge and bioconda, set in such a way that conda-forge takes precedence or flexible priority.

I'll consult @conda-forge/core here. What's the consensus? Is it ok for a different package to "shadow" a bioconda package, considering all the above?

File: recipes/apoc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "apoc" %}
```
Comment (jaimergp) [inline recipes/apoc/meta.yaml:1]: Ok so I spoke with the team and basically we need to consider `bioconda` and `conda-forge` a single unified namespace. That means we can't use `apoc` here. I suggest using the following, but feel free to suggest something else, @haesleinhuepf!

```suggestion
{% set name = "apoc-backend" %}
```

File: recipes/apoc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - scikit-learn >=0.24.2
+
+test:
+  imports:
+    - apoc
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - oclgrind  # [win]
```
Comment (jaimergp) [inline recipes/apoc/meta.yaml:34]: Can't we just use this one for all platforms?

File: recipes/apoc/meta.yaml (hunk 1 above)
Comment (haesleinhuepf) [reply recipes/apoc/meta.yaml:1]: Hi @jaimergp , this is just the conda package name, right? I don't need to change anything in the python code, correct? If so, I'm fine with that suggestion. 🙂

Comment (jaimergp) [reply recipes/apoc/meta.yaml:1]: Yep, it's only what `conda` sees. The import name itself should be ok.

**Comment by jaimergp** [comment]
@goanpeca can you rename the directory from `apoc` to `apoc-backend` too?

**Comment by jakirkham** [comment]
@conda-forge/bioconda-recipes, do you have thoughts on the naming situation here? 🙂

**Comment by goanpeca** [comment]
>@goanpeca can you rename the directory from apoc to apoc-backend too?

Done @jaimergp and I accidentally pressed the merge button and not the commend button 🤦🏼

File: recipes/apoc/meta.yaml (hunk 2 above)
Comment (goanpeca) [reply recipes/apoc/meta.yaml]: Yes we can!

**Comment by jaimergp** [comment]
all good, the feedstock is here https://github.com/conda-forge/apoc-backend-feedstock

**Pull request merged**